rfc/2019-12-9-Talent-Plan-Course-DDS.md

```diff
@@ -0,0 +1,63 @@
+# Proposal: Talent Plan Courses - Distributed Database System
+
+- Author(s): [Ling Jin](https://github.com/JinLingChristoher)
+- Last updated: 2019-12-11
+- Discussion at: [Incubating Program: Talent Plan Courses - Distributed Database System](https://github.com/pingcap/community/issues/141)
+
+## Abstract
+
+The proposal aims at create a distributed database system course to help more people to have a touch of TiDB. Participants of the course should be able to submit pull requests to TiDB related projects after finishing the course materials.
```

Comment:
> should be able to submit pull requests to TiDB related projects

Is this a tempting or rewarding goal for contributors? I think we should be more than that...  I like your previous version better